src/components/ApplicationGroup/AppGroup.service.ts

```diff
@@ -68,6 +70,31 @@ export const getWorkflows = (envID: string): Promise<WorkflowsResponseType> => {
     })
 }
 
+export const getCIConfigList = (envID: string): Promise<CIConfigListType> => {
+    const pipelineList = []
+    const _filteredworkflow: Map<string, any> = new Map()
+    return Promise.all([
+        getEnvWorkflowList(envID),
+        getCIConfig(envID),
+        getModuleInfo(ModuleNameMap.SECURITY),
+        getModuleConfigured(ModuleNameMap.BLOB_STORAGE),
+    ]).then(([workflow, ciConfig, securityInfo, moduleConfig]) => {
+        workflow.result.workflows.forEach((workFlow) => {
```

Comment:
rename inner or outer variable name as it's causing some confusion.

Reply:
Done